include/stdint.h

```diff
@@ -26,10 +26,34 @@
 #error "Required header stdint.h is not defined for current architectue!"
 #endif
 
-#ifdef _USE_STANDARD_TYPES_STDINT
-
 /* The following sections refer to ISO/IEC 9899:1999 */
 
+/* 7.18.1.1 Exact-width integer types */
+
+typedef __s8 int8_t;
+typedef __s16 int16_t;
+typedef __s32 int32_t;
+typedef __s64 int64_t;
+
+typedef __u8 uint8_t;
+typedef __u16 uint16_t;
+typedef __u32 uint32_t;
+typedef __u64 uint64_t;
+
+/* 7.18.1.2 Minimum-width integer types */
+
+typedef int8_t int_least8_t;
+typedef int16_t int_least16_t;
+typedef int32_t int_least32_t;
+typedef int64_t int_least64_t;
+
+typedef uint8_t uint_least8_t;
+typedef uint16_t uint_least16_t;
+typedef uint32_t uint_least32_t;
+typedef uint64_t uint_least64_t;
+
+#ifdef _USE_STANDARD_TYPES_STDINT
```

Comment:
To fully conform with idea of this define, this ifdef should be above  `/* 7.18.1.1 Exact-width integer types */`

in fact C doesn't require to have uint(8/16/32/64)_t types defined.

It's minor as we don't want to support weird architectures but would be great to preserve idea here on wipe it out :)